g9frontend/src/pages/UserPage/UserPageStyle.css

```diff
@@ -155,7 +155,7 @@ html, body {
     display: flex;
     justify-content: center;
     align-items: center;
-    height: 100vh; /* Optional: Set a height to center vertically within the viewport */
+    height: 60vh; /* Optional: Set a height to center vertically within the viewport */
```

Comment:
You should center it to be (vertically) at the center of the **left container**. Sorry if I wasn't specific enough. It's not exactly centered right now and there is too much space between the text and the 'Craft your post' button.